whereby/browser-sdk
# Allow testing React.StrictMode locally

In order to allow for testing components in StrictMode, Storybook is
upgraded to the latest version. Thing is, the latest also fails to
render such stories, and disabling documentation is required to make it
work.

Regardless, upgrading Storybook is maybe not a bad thing, so letting
this stay.

**Tested like**
1. `STORYBOOK_ROOM=<your room url> yarn storybook`
2. Visit story "Room connection only"
3. Verify you get "Connecting..." output, and no Storybook compile error.
